custom_components/sat/config_flow.py

```
@@ -157,7 +162,11 @@ async def async_step_mosquitto_ems(self, _user_input: dict[str, Any] | None = No
 
             return await self.async_step_sensors()
 
-        return self._create_mqtt_form("mosquitto_ems", "ems-esp")
+        default_device = None
+        if not self.data.get(CONF_DEVICE):
```

Ruff (0.8.2)

`CONF_DEVICE` may be undefined, or defined from star imports

(F405)